### ISSUE TITLE
Array is destructured in new beforeAutocompleteSources hook

### ISSUE THREAD

**Issue by peterjaap** [algolia/algoliasearch-magento-2#633]
Also see issue in Magento 1, we have the same issue in Magento 2; https://github.com/algolia/algoliasearch-magento/issues/1076

I tried using the `arguments` object, but returning it proved unsuccesful;

```
algolia.registerHook('beforeAutocompleteSources', function () {
 /// do stuff
    return arguments;
});
```

It gives me an error in _utils.js and _typeahead.js.

**Comment by JanPetr** [comment]
Hello @peterjaap,

the fix has been merged in #641 and released in 1.9.1 version of the extension.

Let us know how it works for you :) Thanks!

**Comment by peterjaap** [comment]
Great thanks!
